### PR TITLE
Notif panel

### LINKED ISSUE #1738: Multiple Selects for Notifications
[LInk to figma > ](https://www.figma.com/file/Goequb9eZDL6WxK7T74oxS/Notifications-(Copy)?node-id=839%3A99688)

- [ ] Checkboxes/ groups of notifications
- [ ] Floating panel in notification centre (bottom of the page)

### PR DESCRIPTION
**Pull request by eldiegod**
fixes: #1738 and by mistake #1737

### CODE REVIEW

**Comment by drillprop** [comment]
If it also fixes #1737 you should assign yourself to this task.  Someone could start working on it by accident. Just saying :P

Anyway, I assigned you and moved these issues on zenhub to `review`

**Comment by eldiegod** [comment]
> If it also fixes #1737 you should assign yourself to this task. Someone could start working on it by accident. Just saying :P
> 
> Anyway, I assigned you and moved these issues on zenhub to `review`

yep u right

File: packages/atlas/src/components/_notifications/NotificationsWidget/NotificationsWidget.tsx
Comment (rafalpawlow) [inline packages/atlas/src/components/_notifications/NotificationsWidget/NotificationsWidget.tsx]: Couldn't we create function for that in `useNotifications` to not repeat this mapping every time?

**Review by rafalpawlow — CHANGES_REQUESTED** [review]
1.  something is wrong with dimensions, in figma it has 613px

Zrzut ekranu 2022-04-8 o 10 58 53

Zrzut ekranu 2022-04-8 o 11 03 24

2. This button is egg-shaped
Zrzut ekranu 2022-04-8 o 10 59 12

3. You forgot about RWD

File: packages/atlas/src/components/_notifications/NotificationsWidget/NotificationsWidget.tsx
Comment (eldiegod) [reply packages/atlas/src/components/_notifications/NotificationsWidget/NotificationsWidget.tsx]: now you can pass notifications obj directly

**Comment by eldiegod** [comment]
> something is wrong with dimensions, in figma it has 613px

Hey @toiletgranny, could you help out a bit in here, I'm not sure how to get the styles to match the width in figma can you tell me what I'm missing?

**Comment by toiletgranny** [comment]
As this was prepared by @TCzechowski, Tomek, can you have a look at Diego's question above?

**Comment by TCzechowski** [comment]
Hi @eldiegod! I don't know what happened in Figma, but the close button in Figma got stretched so the dimensions were off. I fixed it in Figma, but it still didn't match your width. I investigated it a bit and it's probably because you have used the Medium size of the buttons, and I've used Large variant for >MD breakpoints :)

**Comment by eldiegod** [comment]
> hat happened in Figma, but the close button in Figma got stretched so the dimensions were off. I fixed it in Figma, but it still didn't match your width. I investigated it a bit and it's probably because you have used the Medium size of the buttons, and I've used Large variant for >MD breakpoints :)

yep, that solves it, thanks a lot @TCzechowski ! 

also @rafalpawlow the notification tiles might look a bit off from the design rn but that's because of the other notification PR that got merged today Bartosz is working on a fix for that

**Comment by drillprop** [comment]
> also @rafalpawlow the notification tiles might look a bit off from the design rn but that's because of the other notification PR that got merged today Bartosz is working on a fix for that

fixed in #2476

**Comment by kdembler** [comment]
@eldiegod Needs a small rebase. Also what's the status here? Waiting for review from @rafalpawlow?

**Comment by eldiegod** [comment]
> base. Also what's the status here? Waiting for review

@kdembler yes, pretty much

File: packages/atlas/src/views/notifications/NotificationsView.tsx
Comment (rafalpawlow) [inline packages/atlas/src/views/notifications/NotificationsView.tsx]: According to figma:
```suggestion
    <Button variant="tertiary" size={smMatch ? 'large' : 'medium'} onClick={unselectAllNotifications} iconOnly icon={<SvgActionClose />} />
```

**Review by rafalpawlow — CHANGES_REQUESTED** [review]
Looks good, however there is a problem with bottom nav. I think we could do the same I did for cookies banner https://github.com/Joystream/atlas/commit/dc10f63055a2e93b37746dc251dd8dad5d4cf1b0#diff-b2c59b011fdd1a962bf69babef2b7535bb64a75d864f0d921c162c4689246b04R26 but let's confirm this with @TCzechowski 

Zrzut ekranu 2022-04-13 o 18 03 08

**Comment by TCzechowski** [comment]
> Looks good, however there is a problem with bottom nav. I think we could do the same I did for cookies banner [dc10f63#diff-b2c59b011fdd1a962bf69babef2b7535bb64a75d864f0d921c162c4689246b04R26](https://github.com/Joystream/atlas/commit/dc10f63055a2e93b37746dc251dd8dad5d4cf1b0#diff-b2c59b011fdd1a962bf69babef2b7535bb64a75d864f0d921c162c4689246b04R26) but let's confirm this with @TCzechowski
> 
> Zrzut ekranu 2022-04-13 o 18 03 08

Yes, please do the same treatment here @rafalpawlow :)

**Comment by eldiegod** [comment]
@rafalpawlow the problem is your PR is merged to dev, we don't have `useBottomNavStore` here, I added the fixes and commented them until either we rebase rhodes or we merge rhodes

**Review by rafalpawlow — CHANGES_REQUESTED** [review]
Something very bad has happened here 😛

Zrzut ekranu 2022-04-15 o 13 05 38

**Comment by eldiegod** [comment]
> Something very bad has happened here 😛
> 
> Zrzut ekranu 2022-04-15 o 13 05 38

@rafalpawlow oops forgot something, fixed now!

**Comment by rafalpawlow** [comment]
@eldiegod `useBottomNavStore` is already on `rhodes`

**Comment by eldiegod** [comment]
> @eldiegod `useBottomNavStore` is already on `rhodes`

@rafalpawlow  its in there

**Comment by rafalpawlow** [comment]
@eldiegod oh, really sorry I should probably check it before write comment 😅

**Review by rafalpawlow — APPROVED** [review]

**Pull request merged**
